modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -4959,10 +4953,12 @@ SUBROUTINE FillOutputAry(p_FAST, y_FAST, IfWOutput, OpFMOutput, EDOutput, ADOutp
       END IF            
       
       IF ( y_FAST%numOuts(Module_AD) > 0 ) THEN
-         indxLast = indxNext + SIZE(ADOutput) - 1
-         OutputAry(indxNext:indxLast) = ADOutput
-         indxNext = IndxLast + 1
-      END IF
+         do i=1,SIZE(y_AD%Rotors)
+            indxLast = indxNext + SIZE(y_AD%Rotors(i)%WriteOutput) - 1
```

Comment:
If `size(y_AD%Rotors)` is ever more than 1, we should also put an `if (allocated(y_AD%rotors(i)%WriteOutput)` around these calculations before using the size function. If one rotor has no outputs but others do, it could cause a seg fault otherwise.

Reply:
Good point. I might as well add that now.

I had initially skipped such checking here since there are a lot of other places in the glue code that will need updates for more than one rotor.

Reply:
Yeah, that will be a fun job later, too. :) Since this one actually uses a loop on the size of the rotors now, it caught my attention.